### DISCUSSION TITLE
ch-47S cant sling

### DISCUSSION THREAD

**Discussion by MacTheGoon** [Vdauphin/HeartsAndMinds#1342] [Question and answer] [answered]
Good day,
I am currently play testing an HM mission and everything is working as expected, however the RHS CH-47 cannot sling load wrecks.  I thought maybe it was an issue with the RHS version however it appears to do the same with the vanilla version.  Any thoughts?

**Comment by Vdauphin** [comment, marked as answer]
Hello,
Read this: http://vdauphin.github.io/HeartsAndMinds/InGame-documentation#sling-loading
Cheers

**Reply by MacTheGoon** [reply]
Yes, thank you.  I have read it and have used the slingload/towing functionality for a long time now - it just seems as though there is something preventing it now.

**Reply by Vdauphin** [reply]
![20220330180205_1](https://user-images.githubusercontent.com/14364400/160882050-b8052150-3c2e-4e02-83c7-77c4a7a7e2b6.jpg)
![20220330181348_1](https://user-images.githubusercontent.com/14364400/160882052-d4d8056e-f76f-4f22-af01-96ad097aa9e7.jpg)

**Reply by MacTheGoon** [reply]
Yes, i've figured it out and my apologies!
